weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -1058,6 +1134,19 @@ def _insert_call(self, ch_call: CallCHInsertable) -> None:
         if self._flush_immediately:
             self._flush_calls()
 
+    def _insert_calls_delete(
```

Comment:
this looks identical to `_insert_call` except that it supports a list. The column stuff should work just fine due to the Null defaults. I'd recommend just changing `_insert_call` to handle a list

Reply:
The issue is how to manage `all_call_delete_columns` vs. `all_call_insert_columns` which can be different. Not ~ quite ~ identical... @tssweeney